snuba/consumers/consumer_builder.py

```diff
@@ -132,6 +134,8 @@ def __init__(
         self.metrics = metrics
         self.max_batch_size = max_batch_size
         self.max_batch_time_ms = max_batch_time_ms
+        self.max_insert_batch_size = max_insert_batch_size
+        self.max_insert_batch_time_ms = max_insert_batch_time_ms
```

Comment:
Should these default to the original max_batch_size/max_batch_time values?

Reply:
yeah that fallback happens later in the strategy factory